recipe/meta.yaml

```diff
@@ -31,7 +31,7 @@ requirements:
     - {{ compiler('c') }}
```

Comment:
As we do not build cross-platform these lines can be removed from `build`:
```
    - python                                 # [build_platform != target_platform]
    - cffi                                   # [build_platform != target_platform]
```

Reply:
addressed